x-pack/plugin/build.gradle

```diff
@@ -112,6 +112,7 @@ testClusters.all {
   setting 'indices.lifecycle.history_index_enabled', 'false'
   keystore 'bootstrap.password', 'x-pack-test-password'
   keystore 'xpack.security.transport.ssl.secure_key_passphrase', 'testnode'
+  setting 'xpack.searchable.snapshot.shared_cache.size', '10mb'
```

Comment:
More of a comment/note: One somewhat unfortunate side-effect of this change is that we now don't have any REST test coverage for the `0b` cache size use case right?
Should we make sure that a node with `0b` cache size continues to properly start up in some form still? (It was quite helpful to have this coverage in my recent work)

Reply:
I'm not sure if that functionality needs a REST test. As we already start up integration tests that way, that might give us good enough coverage?